web/src/user/user-settings/mfa/MFADevicesPage.ts

```diff
@@ -18,17 +17,8 @@ import { until } from "lit/directives/until.js";
 import { AuthenticatorsApi, Device, UserSetting } from "@goauthentik/api";
 
 export function stageToAuthenticatorName(stage: UserSetting): string {
-    switch (stage.component) {
```

Comment:
I don't think this should be removed, rather have the `stage.title` check at the top and if that's not set, then fallback to the current switch statement...allthough with the logic above there's no real way that stage.title could be empty I suppose, so maybe it should be set to required in the API schema

Reply:
I am happy with either, but I think the setting it required in the schema makes more sense in order to avoid duplication of strings (both in frontend and backend) and also so that we can guarantee the same experience for API users.

Reply:
Actually, the `title` attribute was already required in the schema (because the serializer has no default and is not readonly) but I added an explicit required kwarg anyway,